src/service/layers-impl.js

```diff
@@ -46,6 +46,11 @@ export let SizeDef;
  */
 export let PositionDef;
 
+/**
+ * @typedef {Object<string, *>}
```

Comment:
"Anonymous state/cache passed from ancestry iterator."

When I first read this I thought it was "state of ancestry".